.gitignore

```diff
@@ -3,3 +3,5 @@
 .vscode/
 *audio_frontend*
 *google*
+*__pycache__*
```

Comment:
should pyc file be included as well?

Reply:
The pyc files are all within the __pycache__ directory.